SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -125,4 +135,6 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 	Worker_RequestId NextRequestId = 0;
 
 	ESpatialConnectionType ConnectionType = ESpatialConnectionType::Receptionist;
+	
+	LoginTokenRes_Callback    LoginTokenResCallback = nullptr;
```

Comment:
Should not need to set this to nullptr. This will construct the TFunction with nullptr but this is somewhat hidden and it appears as if you are setting the TFunction value itself to nullptr which on the surface is slightly confusing.

Reply:
I don't think this has been resolved.